src/ServicePulse.Host/vue/src/assets/main.css

```diff
@@ -685,3 +693,13 @@ div.alert.alert-warning strong {
   margin: 25px 0 0;
   height: unset;
 }
+
+/* TODO: why do these have to be !important? */
```

Comment:
I don't understand why we need these extra styles?

Reply:
the classes are used by the templates, as ported from the angularJS code, and without them there are missing style behaviours. These originally lived in particular.css